contrib/terraform/openstack/modules/compute/templates/cloudinit.yaml

```diff
@@ -0,0 +1,16 @@
+#cloud-config  # yamllint disable-line rule:require-starting-space
```

Comment:
I need some hint how to disable starting space in yamllint guys:
Should it be:
`# yamllint disable-line rule:comments.require-starting-space`
Or even:
`# yamllint disable-line rule:comments`

Reply:
```
# yamllint disable-line
#cloud-config
## in some cases novnc console access is required
## it requires ssh password to be set
# yamllint disable-line
#ssh_pwauth: yes
# yamllint disable-line
#chpasswd:
#  list: |
#    root:secret
#  expire: False

## in some cases direct root ssh access via ssh key is required
# yamllint disable-line
#disable_root: false

## in some cases additional CA certs are required
# yamllint disable-line
#ca-certs:
#  trusted: |
#      -----BEGIN CERTIFICATE-----
```

Looks like this works, but strangly I can't manage to make `rule:comments.require-starting-space` to works 🤷